core/templates/components/shared-component.module.ts

```diff
@@ -38,10 +44,12 @@ import { NgbModalModule } from '@ng-bootstrap/ng-bootstrap';
   imports: [CommonModule, MaterialModule, NgbModalModule],
   declarations: [BackgroundBannerComponent, TranslatePipe,
     SharingLinksComponent, ExplorationEmbedButtonModalComponent,
-    SkillMasteryViewerComponent],
+    SkillMasteryViewerComponent, AttributionGuideComponent,
+    LazyLoadingComponent, LoadingDotsComponent],
   entryComponents: [BackgroundBannerComponent, SharingLinksComponent,
     // These elements will remain here even after migration.
-    ExplorationEmbedButtonModalComponent, SkillMasteryViewerComponent
+    ExplorationEmbedButtonModalComponent, SkillMasteryViewerComponent,
```

Comment:
Do you want to address Srijan's comment in your previous PR here?

Reply:
https://github.com/oppia/oppia/pull/9757#discussion_r451380466

Reply:
Yes, just fixing that